packages/jest-runner/fuzz.ts

```diff
@@ -53,7 +53,16 @@ export const fuzz: FuzzTest = (title, fuzzTest) => {
 
 	if (fuzzingConfig.dryRun) {
 		g.describe(title, () => {
-			corpus.inputPaths().forEach(([name, path]) => {
+			const inputsPaths = corpus.inputsPaths();
+			// Mark fuzz tests with empty imputs as skipped to suppress
+			// Jest error.
+			if (inputsPaths.length === 0) {
+				g.test.skip(title, () => {
```

Comment:
Can we add a test for the skipping functionality? Or would it need too much effort?
E.g., the fuzzing function has a side effect (changing a global variable, or touching a temporary file), and the test checks for the presence/absence of the side effect.

Reply:
I will provide the tests in a dedicated PR.